CRM/Event/BAO/Event.php

```diff
@@ -187,15 +187,23 @@ public static function del($id) {
     $groupTree = CRM_Core_BAO_CustomGroup::getGroupDetail(NULL, NULL, $extends);
     foreach ($groupTree as $values) {
       $query = "DELETE FROM " . $values['table_name'] . " WHERE entity_id = " . $id;
-
-      $params = array(
-        1 => array($values['table_name'], 'string'),
-        2 => array($id, 'integer'),
-      );
-
```

Comment:
I agree that these lines do nothing - but it seems like the intent was to replace the unescaped variables in the SQL query.  I think the better option would be to do so.